react/features/conference/components/web/Subject.js

```diff
@@ -29,15 +29,112 @@ type Props = {
     /**
      * Indicates whether the component should be visible or not.
      */
-    _visible: boolean
+    _visible: boolean,
+
+    /**
+     * The JitsiConference from which stats will be pulled.
+     */
+    conference: Object,
+
+    /**
+     * Dominant speaker id
+     */
+    id: string
```

Comment:
you can mark it as local var with unserscore.
```suggestion
    _id: string
```